### PR TITLE
Update `ToggleSwitch` add `OffContent` and  `OnContent`

### PR DESCRIPTION
**Pull request by zh3305**
## Pull request type

Please check the type of change your PR introduces:

- [x] Update
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes

## What is the current behavior?
动画

### CODE REVIEW

**Comment by LHTXQ** [comment]
This change decision is unreasonable, content should be driven by binding variables instead of setting content separately for both states.

**Comment by zh3305** [comment]
> This change decision is unreasonable, content should be driven by binding variables instead of setting content separately for both states.
I don't want to bind too many variables to handle this switch change, and I don't want to handle this unique change by converting when there are multiple states such as switch, split, and close, which makes the code bloated. I looked at the WinUI 3 Gallery code and it solved my problem perfectly. With this change you can still use the Content property to set the label, OffContent OnContent is just an option

动画

**Comment by LHTXQ** [comment]
> > This change decision is unreasonable, content should be driven by binding variables instead of setting content separately for both states.
> 
> 
> I don't want to bind too many variables to handle this switch change, and I don't want to handle this unique change by converting when there are multiple states such as switch, split, and close, which makes the code bloated. I looked at the WinUI 3 Gallery code and it solved my problem perfectly. With this change you can still use the Content property to set the label, OffContent OnContent is just an option
> 
> 动画
> 
> 

🤔行吧……

File: src/Wpf.Ui/Controls/ToggleSwitch.cs [hunk 1]
```diff
@@ -15,4 +16,23 @@ namespace Wpf.Ui.Controls;
 [ToolboxBitmap(typeof(ToggleSwitch), "ToggleSwitch.bmp")]
 public class ToggleSwitch : System.Windows.Controls.Primitives.ToggleButton
 {
+    public static readonly DependencyProperty OffContentProperty = DependencyProperty.Register(
+        "OffContent", typeof(object), typeof(ToggleSwitch), new PropertyMetadata(null));
+
+    [Bindable(true)]
+    public object OffContent
```
Comment (pomianowski) [inline src/Wpf.Ui/Controls/ToggleSwitch.cs:23]: Missing docs

**Review by pomianowski — APPROVED** [review]

**Review by pomianowski — APPROVED** [review]

**Review by pomianowski — APPROVED** [review]

**Review by pomianowski — APPROVED** [review]

**Pull request merged**